skyportal/models/instrument.py

```diff
@@ -89,6 +89,12 @@ class Instrument(Base):
         doc='List of filters on the instrument (if any).',
     )
 
+    sensitivity_data = sa.Column(
+        JSONB,
+        nullable=True,
+        doc="JSON describing the filters on the instrument and the filter's corresponding limiting magnitude and exposure time.",
+    )
+
```

Comment:
You know, in a perfect world, this would have been a table keyed to `Instrument` and `Filter`. But that may be a little too complicated for what we want...

Reply:
https://www.youtube.com/watch?v=RUi54JTgL5s&ab_channel=SimplePlan